node-lib/src/regtest_options.rs

```diff
@@ -61,4 +61,8 @@ pub struct ChainConfigOptions {
     /// PoS NetUpgrade override after Genesis
     #[clap(long)]
     pub chain_pos_netupgrades: Option<bool>,
+
+    /// PoS Genesis staking settings
+    #[clap(long)]
+    pub chain_genesis_staking_settings: Option<String>,
```

Comment:
I believe that if you create a type for the settings, like `GenesisSTakingSettings` and use it here, it will work as long as you provide `FromStr`. May be wrong though.

Reply:
Nice... I didn't know you could create your own types. Done